Sensor_front front_sensor
Sensor_left left_sensor
Sensor_right right_sensor  